README.md

```diff
@@ -97,13 +97,22 @@ Capybara.app = MyRackApp
 
 If you need to test JavaScript, or if your app interacts with (or is located at)
 a remote URL, you'll need to [use a different driver](#drivers).  If using Rails 5.0+, but not using the Rails system tests from 5.1, you'll probably also
-want to swap the "server" used to launch your app to Puma in order to match Rails defaults.
+want to swap the "server" used to launch your app to Puma or thin in order to match Rails defaults.
```

Comment:
`thin` is not the Rails default

Reply:
Seems I did not understand the meaning of the sentence correctly.